packages/dds/tree/src/test/domains/json/benchmarks.ts

```diff
@@ -12,8 +12,8 @@ export function sum(cursor: ITreeCursorNew): number {
         total += value;
     }
 
-    for (let moreFields = cursor.firstField(); moreFields; moreFields = cursor.nextField()) {
-        for (let inField = cursor.firstNode(); inField; inField = cursor.nextField()) {
+    for (let inField = cursor.firstField(); inField; inField = cursor.nextField()) {
```

Comment:
Switching to the firstField/nextField approach revealed this bug. I figured it was small and semi-related enough that it could go in as part of this PR but let me know if you want me to separate it out.

Reply:
It's no longer related so I'll open a separate PR.